services/skus/controllers.go

```diff
@@ -1590,14 +1590,10 @@ func handleReceiptErr(err error) *handlers.AppError {
 	}
 
 	switch {
-	case errors.Is(err, errPurchaseFailed):
-		result.ErrorCode = purchaseFailedErrCode
+	case errors.Is(err, errPurchaseExpired):
+		result.ErrorCode = purchaseExpiredErrCode
```

Comment:
note: I checked the skus SDK and searched the Brave github org and could not see anything depending on this error code so it looks safe to change/add.